README.md

```diff
@@ -83,11 +83,45 @@ require"telescope".setup {
       -- Max number of authors to write in the formatted citation
       -- following authors will be replaced by "et al."
       citation_max_auth = 2,
+      -- Use context awareness by default
+      context = false
     },
   }
 }
 ```
 
+### Context Aware Bibliography File
+
+Two common contexts are included:
```

Comment:
Maybe its just me, but it took a while to understand what context aware means. Maybe add something like: 'If enabled, the plugin will look for context lines in your currently opened file that imply which bibliography file you want to use. See bellow for common examples based on filetype:'

Reply:
Fixed in latest commit